server/lib/date-adjust.js

```diff
@@ -0,0 +1,17 @@
+"use strict";
+
+// Require fs to write to initial-tweets.json
+const fs = require("fs");
+// Requiring a JSON file automatically parses it and returns the data.
+const tweetsJSON = require('../data-files/initial-tweets.json');
+
+// Write recent dates to initial-tweets.json
+// Specifically sync to not interfere with student functions
+module.exports = () => {
+  // Iterate tweets and convert the previous time to now()...
+  // Then subtract one day (1000ms * 60sec * 60min * 24hrs) * (tweets.length - index)
+  // This keeps the newest tweets at the bottom, and allows for further tweets to be added.
+  tweetsJSON.map((tweet, index) => tweet.created_at = Date.now() - (1000 * 60 * 60 * 24 * (tweetsJSON.length - index)));
```

Comment:
I think you might have reverted too far. The magic numbers are back.

Reply:
Can we also change this to `tweetsJSON = tweetsJSON.map...` so that it's evident the original array is being mutated?

Reply:
Okay so I did some research on what I thought magic numbers are, which are hard to read esoteric numerical declarations in code - so I've declared `oneDayMs` as 
```
// 1000 milliseconds * 60 seconds * 60 minutes * 24 hours
  const oneDayMs = 1000 * 60 * 60 * 24
```
Ref this [Stack Overflow exchange](https://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad).

I could also use Date.setDay() and various other methods but that might complicate matters by converting epoch time to a Date object, then doing the math required and converting it back to epoch time.  This imo would complicate things so please let me know what direction you'd like me to go in :)